### PR TITLE
Fixed a bug in EVA to NPA translation code

### PR DESCRIPTION
**Pull request by drichmond**
* dram_x_min/max assumed that an origin was passed.

### CODE REVIEW

File: libraries/bsg_manycore_eva.cpp [hunk 1]
```diff
@@ -396,28 +400,31 @@ static int default_eva_get_x_coord_dram(const hb_mc_manycore_t *mc,
                                         const hb_mc_eva_t *eva,
                                         hb_mc_idx_t *x) {
         hb_mc_coordinate_t pod = hb_mc_config_pod(cfg, *src);
-#ifdef DEBUG
+        hb_mc_coordinate_t og = hb_mc_config_pod_vcore_origin(cfg, pod);
+#ifndef DEBUG
```
Comment (mrutt92) [inline libraries/bsg_manycore_eva.cpp:404]: fix

File: examples/library/test_printing/main.c [hunk 2]
```diff
@@ -50,6 +50,7 @@
 #include <time.h>
 #include <stdlib.h>
 #include <bsg_manycore.h>
+#include <bsg_manycore_printing.h>
```
Comment (mrutt92) [inline examples/library/test_printing/main.c:53]: why is this here?

File: libraries/bsg_manycore_eva.cpp (hunk 1 above)
Comment (drichmond) [reply libraries/bsg_manycore_eva.cpp]: Oops, thought I undid this. Fixed

File: examples/library/test_printing/main.c (hunk 2 above)
Comment (drichmond) [reply examples/library/test_printing/main.c:53]: Fails silently with a linker error if it isn't.

Comment (drichmond) [reply examples/library/test_printing/main.c:53]: I'll pull it out into a separate PR

**Review by mrutt92 — APPROVED** [review]
